UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -194,6 +194,7 @@ ScriptsTool::ScriptsTool() : QWidget(nullptr), ui(new Ui_ScriptsTool)
 		config_get_string(config, "Python", "Path" ARCH_NAME);
 	ui->pythonPath->setText(path);
 	ui->pythonPathLabel->setText(obs_module_text(PYTHONPATH_LABEL_TEXT));
+	showPythonVersion();
```

Comment:
Given that the label is always "shown" (albeit with an empty text"), `updatePythonVersion` (or even `updatePythonVersionLabel`) would be more descriptive of what the function does.

Reply:
Good point. Fixed.